bftengine/src/bftengine/messages/PrePrepareMsg.hpp

```diff
@@ -38,6 +38,7 @@ class PrePrepareMsg : public MessageBase {
     EpochNum epochNum;
     uint16_t flags;
     uint64_t batchCidLength;
+    int64_t time;
```

Comment:
Please set it to 0 in the PrePrepare constructor (we rely on the 0 value to indicate that there is not time record)

Reply:
Thanks @yontyon, fixing this.

Reply:
Another question is, the time is int64_t. So it can take negative values also. Although negative time makes sense in the Minkowski space-time metric in concord it will always be positive. So should we also change all the references:
 if (msg->getTime() == 0)  to  if (msg->getTime() <= 0) 
Since  time_since_epoch() of the clock may return negative values.
https://stackoverflow.com/questions/29436297/c-chrono-giving-negative-epoch-time

Reply:
Discussed offline with @yontyon , he said he is fine with the check for the negative range. So resolving the comment.